WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -1123,6 +1131,26 @@ private void updateWhitelistSettings(int val) {
                 getWhitelistSummary(val));
     }
 
+    private void handleStartOver() {
+        // Only paid plans should be handled here, free plans should be redirected to website from "Start Over" button
+        if (mSite == null || mSite.getHasFreePlan()) {
+            return;
+        }
+        Intent intent = new Intent(Intent.ACTION_SEND);
+        intent.setType("text/plain");
```

Comment:
Thanks @tonyr59h! It still shows NFC applications such as Android Beam, but it's better than before. Fixed in b953027c8fc4d7eff656ca39d7ddd678c090699f